### DISCUSSION TITLE
v2024.3.110

### DISCUSSION THREAD

**Discussion by zyrouge** [zyrouge/symphony#479] [Announcements]
## What's Changed

* Added individual folders view. (#203)
* Added monochrome icon back. (#407)
* Fixed incorrect album songs sort. (#409)
* Added Persian (Farsi) translation. (@alijafari-gd in https://github.com/zyrouge/symphony/pull/414)
* Added Polish translation. (@rehork in https://github.com/zyrouge/symphony/pull/478)
* Updated Portuguese translation. (@GeovaneDev in https://github.com/zyrouge/symphony/pull/428)
* Updated Japanese and Uchinaguchi translations. (@kuragehimekurara1 in https://github.com/zyrouge/symphony/pull/462)

## New Contributors
* @alijafari-gd made their first contribution in https://github.com/zyrouge/symphony/pull/414
* @rehork made their first contribution in https://github.com/zyrouge/symphony/pull/478

**Full Changelog**: https://github.com/zyrouge/symphony/compare/v2024.1.109...v2024.3.110

<hr /><em>This discussion was created from the release <a href='https://github.com/zyrouge/symphony/releases/tag/v2024.3.110'>v2024.3.110</a>.</em>

**Comment by Moriarty-47** [comment]
Please add the feature of deleting files directly from the Symphony

**Comment by devboloji** [comment]
Thanks for the wonderful app and also it showing the live lyrics if there attached lyrics to audio file. Interface is unique. I have an issue. I am using my phone with largest "screen size" and "font-size" at maximum in my android which is stock android with 6.55inch display. In the app if a pop-up I opened it's not closing by back button, to close that pop-up I should click anywhere on screen to get lose of pop-up. For example, if I go to app settings> interface> "for you" and I click it, it shows as pop up and after modifying the sorting I cannot get out from that popup. Back button won't work ( I am using 3-button navigation ). to temporarily fix is, I completely need to quit that app and open again, there is no way to close at all. It's not a bigger issue can manage it that's why I am reporting here. I am looking for this kind of app features for 6 years. Thanks for app. will contribute or donate within month or two. 🥳

**Reply by zyrouge** [reply]
Patch for this was released in nightly builds. Let me know if you face it in that. Thanks for those kind words! :)

**Reply by devboloji** [reply]
I am looking for sleep timer in this app. i couldn't find it! The feature is not available?

**Reply by zyrouge** [reply]
Now playing page -> vertical dots on right bottom -> sleep timer.

**Reply by devboloji** [reply]
> Now playing page -> vertical dots on right bottom -> sleep timer.

![Screenshot_20240625-225556_Symphony](https://github.com/zyrouge/symphony/assets/87069967/84678195-ef98-4ad8-9302-50df9498fdc7)


I couldn't find it due to dpi issues. but if I click on edge of the screen at the vertical right bottom it appears (sleep timer) thanks but what could be idea to fit that exceeded button on screen if user using small display or user like me?? 
I can change the screen size and font size in the app settings. That's a great thing, everybody should add this display  size and font size for their apps, but can we fix those without changing those both?  any idea?

**Reply by zyrouge** [reply]
Maybe truncating "Playing ... of ..." to "... / ..." on smaller devices and even moving those icons into the menu can be done. Open an issue for this to be tracked.

**Comment by 8lec-R** [comment]
Love the app. Its wonderful. Was wondering how hard would it be to have gapless playback implemented. I really miss this feature

**Reply by zyrouge** [reply]
Identify how long the "silent" part is will be the hardest part. Symphony does not parse audio data, so getting it will be expensive.

**Reply by 8lec-R** [reply]
I use opus files on my phone, afaik opus natively supports gapless playback. No need for audio processing. If you like I can help you test for formats that natively support gapless playback such as opus and flac

**Reply by zyrouge** [reply]
> I use opus files on my phone, afaik opus natively supports gapless playback. No need for audio processing. If you like I can help you test for formats that natively support gapless playback such as opus and flac

As I mentioned, if it is part of the spec, it should already be working. That said, let me know if there are any such behavior.

**Reply by 8lec-R** [reply]
> > I use opus files on my phone, afaik opus natively supports gapless playback. No need for audio processing. If you like I can help you test for formats that natively support gapless playback such as opus and flac
> 
> As I mentioned, if it is part of the spec, it should already be working. That said, let me know if there are any such behavior.

Ok. I tried extensively with many different codecs and formats. Even different devices. I primarily use OPUS and Flac and they both definitely do support gapless playback as part of their spec but in Symphony it doesn't work, there's always a hiccup between different files. Headphones, Bluetooth or speakers, all are the same.

I haven't tried any of the alpha builds so idk if it's been fixed in those. Let me know if that's the case, I can install it on a secondary device to test

**Reply by zyrouge** [reply]
I'd have to check if Android supports it. Will do as part of #140.
